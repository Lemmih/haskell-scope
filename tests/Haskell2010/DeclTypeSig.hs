{-# LANGUAGE Haskell2010 #-}
module DeclTypeSig where

a :: ()
a = ()

b, c, d :: ()
(b,c,d) = ()
