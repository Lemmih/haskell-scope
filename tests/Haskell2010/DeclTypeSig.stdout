Scope errors:

Definitions:
  Definition 1, from DeclTypeSig.decl1:
    [4ma[0m = ()
  Definition 2, from DeclTypeSig.decl3:
    ([4mb[0m,c,d) = ()
  Definition 3, from DeclTypeSig.decl3:
    (b,[4mc[0m,d) = ()
  Definition 4, from DeclTypeSig.decl3:
    (b,c,[4md[0m) = ()

Use sites:
  Definition used: 1
    [4ma[0m :: ()
  Definition used: 2
    [4mb[0m, c, d :: ()
  Definition used: 3
    b, [4mc[0m, d :: ()
  Definition used: 4
    b, c, [4md[0m :: ()
